services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/CreateHankeRequest.kt

```diff
@@ -140,7 +140,7 @@ data class NewHankealue(
     )
     override val tarinaHaitta: Haitta13? = null,
     @field:Schema(
-        description = "Area name",
+        description = "Area name, must not be null or empty",
```

Comment:
If the field must be non-null, the type should also be non-nullable.

Reply:
Since all other fields are nullable in this class, I thought that this should be as well. E.g. 
```
    @field:Schema(
        description = "Nuisance start date, must not be null",
        maximum = "2099-12-31T23:59:59.99Z",
    )
    override val haittaAlkuPvm: ZonedDateTime? = null,
```

Reply:
This tasks ticket mentions that a hankealue must have a name in order to be made public. Hence, this field is nullable. I guess the description could be updated to something like "required for hanke to be public" etc.

Reply:
Made hankealue.nimi mandatory always and everywhere.